zospy/analyses/mtf.py

```diff
@@ -237,3 +237,125 @@ def fft_through_focus_mtf_fromcfg(
     )
 
     return analysis.complete(oncomplete, result)
+
+
+def huygens_mtf(
+    oss: OpticStudioSystem,
+    pupilsampling: str | int = "32x32",
+    imagesampling: str | int = "32x32",
+    imagedelta: float = 0.0,
```

Comment:
```suggestion
    image_delta: float = 0.0,
```

See previous comment; this name is also used in [psf.huygens_psf](https://github.com/MREYE-LUMC/ZOSPy/blob/8b0d401870c36541df7117ebb18cdb764026740b/zospy/analyses/psf.py#L16).